enterprise/suppress-user/internal/memory/memory.go

```diff
@@ -1,13 +1,16 @@
 package memory
 
 import (
+	"errors"
 	"io"
 	"sync"
 
 	"github.com/rudderlabs/rudder-go-kit/logger"
 	"github.com/rudderlabs/rudder-server/enterprise/suppress-user/model"
 )
 
+var ErrNoValueFound = errors.New("no value found")
```

Comment:
We can move this to enterprise/suppress-user/model/model.go and use it in both badger and memory implementation